Source/WebCore/PAL/pal/PALSwift/CryptoKitShim.swift

```diff
@@ -464,60 +464,170 @@ public enum EdKeyAgreementAlgorithm {
     case x448
 }
 
-public struct EdRv {
-    public var errCode: ErrorCodes = .defaultValue
+public struct EdReturnValue {
+    public var errorCode: ErrorCodes = .defaultValue
     public var signature: OptionalVectorUInt8 = OptionalVectorUInt8()
     public var keyBytes: OptionalVectorUInt8 = OptionalVectorUInt8()
 }
 
 public class EdKey {
-    public static func sign(algo: EdSigningAlgorithm, key: SpanConstUInt8, data: SpanConstUInt8) -> EdRv {
-        var rv = EdRv()
+    public static func sign(algo: EdSigningAlgorithm, key: SpanConstUInt8, data: SpanConstUInt8)
+        -> EdReturnValue
+    {
+        var rv = EdReturnValue()
         do {
             switch algo {
             case .ed25519:
                 let priv = try Curve25519.Signing.PrivateKey(span: key)
                 rv.signature = Cpp.makeOptional(try priv.signature(span: data))
-                rv.errCode = .success
+                rv.errorCode = .success
             case .ed448:
-                rv.errCode = .unsupportedAlgorithm
+                rv.errorCode = .unsupportedAlgorithm
             }
         } catch {
-            rv.errCode = .failedToSign
+            rv.errorCode = .failedToSign
         }
         return rv
     }
-    public static func verify(algo: EdSigningAlgorithm, key: SpanConstUInt8, signature: SpanConstUInt8, data: SpanConstUInt8) -> EdRv {
-        var rv = EdRv()
+    public static func verify(
+        algo: EdSigningAlgorithm, key: SpanConstUInt8, signature: SpanConstUInt8,
+        data: SpanConstUInt8
+    ) -> EdReturnValue {
+        var rv = EdReturnValue()
```

Comment:
It would be good to also rename "rv" to something that's not an abbreviation, like "result"